Shared/Actions/Rich Menu/Models/RMIconContainer.swift

```diff
@@ -0,0 +1,122 @@
+import SwiftUI
+
+/// Container for rendering Icons in Menu Items
+struct RMIconContainer<Icon: View>: View {
+    let icon: Icon
+    var backgroundColor: Color
+    var backgroundShape: RMBackgroundShape
+
+    /// Initializes with a custom view as the icon and the specified background color
+    /// - Parameters:
+    ///   - icon: any view, will be clipped to 93x93
```

Comment:
```suggestion
    ///   - icon: Any view. Will be clipped to 93x93.
```

This style change applies to all comments.